packages/@lwc/rollup-plugin/src/__tests__/fixtures/expected_default_config_simple_app_relative.js

```diff
@@ -0,0 +1,103 @@
+(function (lwc) {
```

Comment:
You might wonder why there's a new snapshot file here. The reason is that the old tests had two separate tests sharing the same snapshot file. It seemed cleaner to me to have two separate snapshot files instead.